setup.py

```diff
@@ -34,7 +34,7 @@ def run(self):
     "datasets>=2.7.0",
 ]
 
-IMAGE_REQUIRE = DATALAB_REQUIRE + ["cleanvision==0.3.2"]
+IMAGE_REQUIRE = DATALAB_REQUIRE + ["cleanvision>=0.3.2,<=0.3.4"]
```

Comment:
```suggestion
IMAGE_REQUIRE = DATALAB_REQUIRE + ["cleanvision>=0.3.2"]
```

Reply:
can we just enforce that all cleanvision releases going forward must work w Datalab nicely?  (just like we enforce all cleanvision releases work well with the cleanvision-examples)

Reply:
Sure